src/main/java/supersymmetry/common/metatileentities/multi/electric/MetaTileEntityFlareStack.java

```diff
@@ -0,0 +1,69 @@
+package supersymmetry.common.metatileentities.multi.electric;
+
+import gregtech.api.capability.impl.MultiblockRecipeLogic;
+import gregtech.api.metatileentity.MetaTileEntity;
+import gregtech.api.metatileentity.interfaces.IGregTechTileEntity;
+import gregtech.api.metatileentity.multiblock.IMultiblockPart;
+import gregtech.api.metatileentity.multiblock.MultiblockAbility;
+import gregtech.api.metatileentity.multiblock.RecipeMapMultiblockController;
+import gregtech.api.pattern.BlockPattern;
+import gregtech.api.pattern.FactoryBlockPattern;
+import gregtech.client.renderer.ICubeRenderer;
+import gregtech.client.renderer.texture.Textures;
+import gregtech.common.blocks.BlockBoilerCasing.BoilerCasingType;
+import gregtech.common.blocks.BlockFireboxCasing;
+import gregtech.common.blocks.MetaBlocks;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.ResourceLocation;
+import supersymmetry.api.capability.impl.NoEnergyMultiblockRecipeLogic;
+import supersymmetry.api.recipes.SuSyRecipeMaps;
+
+import javax.annotation.Nonnull;
+
+import static gregtech.api.util.RelativeDirection.*;
+
+public class MetaTileEntityFlareStack extends RecipeMapMultiblockController {
+    public MetaTileEntityFlareStack(ResourceLocation metaTileEntityId) {
+        super(metaTileEntityId, SuSyRecipeMaps.FLARE_STACK);
+        this.recipeMapWorkable = new MultiblockRecipeLogic(this, true);
+    }
+
+    public MetaTileEntity createMetaTileEntity(IGregTechTileEntity tileEntity) {
+        return new MetaTileEntityFlareStack(this.metaTileEntityId);
+    }
+
+    protected BlockPattern createStructurePattern() {
+        return FactoryBlockPattern.start(FRONT, RIGHT, UP)
+                .aisle(new String[]{"S"})
+                .aisle(new String[]{"P"}).setRepeatable(3,7)
+                .aisle(new String[]{"F"})
```

Comment:
I believe you can remove the `new String[]{}` part as these methods accept varargs.